docs/src/references/mappers/c8y-mapper.md

```diff
@@ -568,6 +568,305 @@ c8y/alarm/alarms/create
 
 </div>
 
+### Data
+
+The data telemetry type can be used to store additional information about the device/child/service. Such information could included; operation system name/version, communication statistics, device status, or any other information that is not suited to measurements, events or alarms.
+
+#### Data - Main device
+
+A device's digital twin model can be updated by publishing to a specific topic.
+
+The type part of the topic is used to group the data so it is easier for components to subscribe to relevant parts.
```

Comment:
Do we need to restrict these to a single level? It definitely makes sense to start with that, but technically, we can support as many sub-levels as needed and for the mapping we can map `data/a/b/c` to the following payload structure, if the cloud only supports a single level:

```json
"a": {
    "b": {
        "c": { // payload... }
}
```

But, that should be a mapper restriction and thin-edge can be more open.